src/core/Akka.Cluster/Configuration/Cluster.conf

```diff
@@ -92,7 +92,16 @@ akka {
     # It has support for https://github.com/dwijnand/sbt-dynver format with `+` or
     # `-` separator. The number of commits from the tag is handled as a numeric part.
     # For example `1.0.0+3-73475dce26` is less than `1.0.10+10-ed316bd024` (3 < 10).
-    app-version = "0.0.0"
+    #
+    # DEFAULT: by default the app-version will default to the entry assembly's version,
+    # i.e. the assembly of the executable running `Program.cs`
+    # 
+    # Values can be "assembly-version" or a version string as defined above, i.e.
+    # app-version = "1.0.0"
+    # app-version = "1.1-beta1"
+    # app-version = "1"
+    # app-version = "1.1"
+    app-version = assembly-version 
```

Comment:
Changed the default value here to `assembly-version`, which activates this assembly-scanning default behavior